packages/dev/inspector-v2/src/components/properties/materials/normalMapProperties.tsx

```diff
@@ -18,8 +27,18 @@ export const NormalMapProperties: FunctionComponent<{ material: MaterialWithNorm
 
     return (
         <>
-            <BoundProperty component={SwitchPropertyLine} label="Invert X Axis" target={material} propertyKey="invertNormalMapX" />
-            <BoundProperty component={SwitchPropertyLine} label="Invert Y Axis" target={material} propertyKey="invertNormalMapY" />
+            {CheckNormalMapPropertyNames(material) && (
+                <>
+                    <BoundProperty component={SwitchPropertyLine} label="Invert X Axis" target={material} propertyKey="invertNormalMapX" />
+                    <BoundProperty component={SwitchPropertyLine} label="Invert Y Axis" target={material} propertyKey="invertNormalMapY" />
+                </>
+            )}
+            {!CheckNormalMapPropertyNames(material) && (
+                <>
+                    <BoundProperty component={SwitchPropertyLine} label="Invert X Axis" target={material} propertyKey="_invertNormalMapX" />
+                    <BoundProperty component={SwitchPropertyLine} label="Invert Y Axis" target={material} propertyKey="_invertNormalMapY" />
+                </>
+            )}

```

Comment:
You can just call the check function once:
```suggestion
            {CheckNormalMapPropertyNames(material) ? (
                <>
                    <BoundProperty component={SwitchPropertyLine} label="Invert X Axis" target={material} propertyKey="invertNormalMapX" />
                    <BoundProperty component={SwitchPropertyLine} label="Invert Y Axis" target={material} propertyKey="invertNormalMapY" />
                </>
            ) : (
                <>
                    <BoundProperty component={SwitchPropertyLine} label="Invert X Axis" target={material} propertyKey="_invertNormalMapX" />
                    <BoundProperty component={SwitchPropertyLine} label="Invert Y Axis" target={material} propertyKey="_invertNormalMapY" />
                </>
            )}
```